src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -2599,6 +2599,31 @@ public static IEnumerable<object[]> VerifyUploadServersStreamsAndExpectedData
             }
         }
 
+        [Fact]
+        public async Task GetAsync_ExpectContinueTrue_NoContent_StillSendsHeader()
+        {
+            if (IsNetfxHandler)
```

Comment:
Why use the 'if' instead of SkipOnTargetFramework(...FullFramework) ?

I know that we use IsCurlHandler, IsWinHttpHandler, UseSocketsHttpHandler in the 'if' because we can't use SkipOnTargetFramework.  But for this that just skips on NETFX, maybe it would be better to use that?

Reply:
Makes sense. Will change.

Reply:
Also, have you tested UAP?  It doesn't support Expect: 100-continue.   Not sure what it does if you manually put that header on it.

I recently got the UAP tests to run clean so I wouldn't want this new test to regress things.  UAP tests don't run in CI right now (only the test source is built). So, you would need to run the tests on your dev box.

Reply:
I'm not set up to run them right now. Would you prefer I mark it as skipped right now on uwp or submit without that?